guides/common/modules/snip_deadlock_warning.adoc

```diff
@@ -0,0 +1,11 @@
+[WARNING]
+====
+{SmartProxy} synchronization might fail with deadlock after upgrading the {Project}.
+To avoid this, ensure that no {SmartProxy} synchronization tasks are in progress and then stop all workers on the {SmartProxy} except one:
```

Comment:
```suggestion
To avoid this, ensure that no {SmartProxy} synchronization tasks are in progress and then stop all workers on the {SmartProxy} except one after the upgrade:
```
or probably even better:
```suggestion
After the upgrade, ensure that no {SmartProxy} synchronization tasks are in progress and then stop all workers on the {SmartProxy} except one:
```
I just think it would be useful to really drive home the point that users must do this after the upgrade. (I know that the whole section is called 'post-upgrade tasks', but still :shrug:)